### DISCUSSION TITLE
How to set the default image size?

### DISCUSSION THREAD

**Discussion by a3nima** [lllyasviel/stable-diffusion-webui-forge#604] [Q&A] [answered]
I haven't found anything in the settings menu. Change default resolution from 512x512 to 1024. I did it in A1111, but cannot remember how. TIA!

**Comment by TheRealPhooney** [comment, marked as answer]
Settings --> Other- Defaults

It's not just for image sizes though, it will save the sampler, the steps, the hiresfix settings, whether you have face restore enabled, etc

**Reply by a3nima** [reply]
Thanks!
